kraken-build/src/kraken/std/python/buildsystem/uv.py

```diff
@@ -337,7 +337,7 @@ def get_path(self) -> Path:
         return self._env_path
 
     def install(self, settings: PythonSettings) -> None:
-        _run_with_uv_indexes(["uv", "sync", "--all-extras"], settings, self.project_directory)
+        _run_with_uv_indexes(["uv", "sync", "--all-extras", "--locked"], settings, self.project_directory)
```

Comment:
Should we remove the `--all-extras` in favor of the `default-groups`? the extra are removed on the first `uv run` anyway as the venv is sync.
```suggestion
        _run_with_uv_indexes(["uv", "sync", "--locked"], settings, self.project_directory)
```

Reply:
I'm happy with this, done!